includes/class-sensei-utils.php

```diff
@@ -1529,38 +1529,45 @@ public static function user_started_course( $course_id = 0, $user_id = 0 ) {
 	 * @param int $user_id   User ID.
 	 * @return int|bool false or comment_ID
 	 */
-	public static function has_started_course( $course_id = 0, $user_id = 0 ) {
-
-		$user_started_course = false;
-
-		if ( $course_id ) {
+	public static function get_course_progress_comment_id( $course_id, $user_id = null ) {
```

Comment:
Should we update the doc comment for this?

Reply:
Fixed! 5a1eac312c884c3c08255586f27d0423705b7e5b